client/my-sites/themes/test/logged-out.jsx

```diff
@@ -108,7 +108,14 @@ describe( 'logged-out', () => {
 	test( 'renders without error when themes are present', async () => {
 		const store = createReduxStore();
 		setStore( store );
-		store.dispatch( receiveThemes( themes, 'wpcom', DEFAULT_THEME_QUERY, themes.length ) );
+		store.dispatch(
+			receiveThemes(
+				themes,
+				'wpcom',
+				{ ...DEFAULT_THEME_QUERY, collection: 'recommended' },
```

Comment:
Should we change the DEFAULT_THEME_QUERY to filter for recommended themes instead?

Reply:
I looked into and ultimately refrained from doing so because `DEFAULT_THEME_QUERY` is used in many places and theme showcase is the only place that is interested in recommended themes.

Actually, `DEFAULT_THEME_QUERY` is not used in the theme showcase at all, I don't know why it's used in this test.